src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq.c

```diff
@@ -78,6 +78,10 @@
 static DEFINE_IDR(xocl_xgq_vmr_cid_idr);
 #define XOCL_VMR_INVALID_CID	0xFFFF
 
+/* Retry is set to 200 seconds for SC to be active*/
```

Comment:
I thought you and @sandeep said that it is about 5 seconds, and the old xmc.c also wait up to 5 seconds. Why this is 200 seconds. In the very worst case, the sc is indeed not ready, the xrt probe will hung up to 200 seconds. This is not acceptable. I would think 5 seconds is the max wait time.

Reply:
Hi David, we discussed about the possible worst case timeout value i.e. 200 seconds in the last meeting with Max, Stephen and Vamshi. It was agreed to add this value in the XGQ driver and display SC not ready error message till it comes online.